<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>1344</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar:vertical {
     border: 1px solid grey;
     background: white;
     width: 40px;
     margin: 41px 0 41px 0;
 }
 QScrollBar::handle:vertical {
     background: lightblue;
     min-height: 40px;
     max-height: 100px;
 }
 QScrollBar::add-line:vertical {
     border: 1px solid grey;
     background: white;
     height: 40px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical {
     border: 1px solid grey;
     background: white;
     height: 40px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     border: 1px solid grey;
     width: 3px;
     height: 3px;
     background: white;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>740</width>
        <height>1589</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="25" column="0">
        <widget class="QCheckBox" name="useLargerFont">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Use larger font in drive slot descriptions</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QComboBox" name="serialPortBaudCombo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>19200 (1x)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400 (2x)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600 (3x)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="serialPortDivisorLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>High speed mode POKEY divisor:</string>
         </property>
        </widget>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>        (Required for AtariDOS compatibility)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QSpinBox" name="writeACKDelayEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QSpinBox" name="serialPortDivisorEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="value">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QCheckBox" name="emulationHighSpeedExeLoaderBox">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Use high speed executable loader</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string> Folder Images:</string>
         </property>
        </widget>
       </item>
       <item row="28" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>SIO port emulation:</string>
         </property>
        </widget>
       </item>
       <item row="22" column="0">
        <widget class="QLabel" name="i18nLanguageLabel">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QCheckBox" name="filterUscore">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Filter out underscore character from file names</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="24" column="0">
        <widget class="QCheckBox" name="saveWinPosBox">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Save window positions and sizes</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="26" column="0">
        <widget class="QCheckBox" name="customDriveNoCheckbox">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Custom drive numbers </string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Emulation settings:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="serialPortHandshakeLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Handshake method:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QComboBox" name="serialPortHandshakeCombo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>RI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DSR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CTS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SOFT</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QSpinBox" name="emulationCustomCasBaudSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>230</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>425</number>
         </property>
         <property name="maximum">
          <number>875</number>
         </property>
         <property name="value">
          <number>600</number>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="serialPortUseDivisorsBox">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Use non-standard speeds</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLineEdit" name="bluetoothNameEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SIO2BT</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QCheckBox" name="emulationUseCustomCasBaudBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Use custom baud rate for cassette emulation</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="serialPortBaudLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>High speed mode baud rate:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="serialPortInterfaceCombo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>SIO2PC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SIO2BT</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QComboBox" name="i18nLanguageCombo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>User interface:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="serialPortInterfaceLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Serial interface:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="bluetoothNameLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Bluetooth name:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="writeACKDelayLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Write ACK delay [ms]</string>
         </property>
        </widget>
       </item>
       <item row="29" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>130</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="27" column="0">
        <widget class="QLineEdit" name="customDriveNo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>230</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1,2,3,4,5,7</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
